tests/unittests/config/test_schema.py

```diff
@@ -702,6 +706,56 @@ def test_validateconfig_file_strictly_validates_schema(
         with pytest.raises(SchemaValidationError, match=error_msg):
             validate_cloudconfig_file(config_file.strpath, schema, annotate)
 
+    @skipUnlessJsonSchema()
+    @responses.activate
+    @pytest.mark.parametrize("annotate", (True, False))
+    @mock.patch("cloudinit.url_helper.time.sleep")
+    @mock.patch(M_PATH + "os.getuid", return_value=0)
+    def test_validateconfig_file_include_validates_schema(
```

Comment:
Can we add one more test case here for checking an `#include` that succeeds?

Reply:
Done.